src/Classes/Factories/HiringPartnerModelFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: imogen
+ * Date: 27/11/2018
+ * Time: 15:45
+ */
```

Comment:
phpstorm autogenerated stuff should be removed

Reply:
Removed in 12d1738